File: TestesAuto/Bild/Login/LoginSteps.java
package bild.Login;

import bild.Utils;
import cucumber.api.java.en.Given;
import cucumber.api.java.en.Then;
import cucumber.api.java.en.When;

public class LoginSteps {

	LoginPage page = new LoginPage();

	@Given("Eu entro na pagina de login")
	public void efetuarLogin() {
		page.efetuarLogin();
		Utils.logPrint("Login","Realizando login no sistema");
	}

	@When("Envio o email e senha do usuario")
	public void entrarPaginaDeLogin(){
		page.enviarEmailESenha();
		Utils.logPrint("Login", "Enviar email e senha do usuário");
	}

	@Then("Eu devo estar logado")
	public void efetuarLogout() {
	}
}

File: TestesAuto/BildPython/001_Login/features/steps/login_steps.py
from behave import *

from ..pages.login_page import login_page

login_page = login_page()

@Given ('Acesso o site "{site}"')
def acessar_site(context, site):
    login_page.acessar_site(site)

@When ('Realizo login com email "{email}" e senha "{senha}"')
def logar(context, email, senha):
    login_page.logar(email, senha)